docs/spark/how-to-guides/connect-to-azure-storage.md

```diff
@@ -12,7 +12,7 @@ ms.custom: mvc,how-to
 
 In this article, you learn how to connect to an Azure Data Lake Storage (ADLS) Gen 2 or Windows Azure Storage Blob (WASB) account through an instance of [.NET for Apache Spark](https://github.com/dotnet/spark) running locally on your Windows machine.
 
-[!INCLUDE [.NET Core 3.1 Warning](../../includes/net-core-31-spark.md)]
+[!INCLUDE [.NET Core 3.1 Warning](./includes/net-core-31-spark.md)]
```

Comment:
You still need to go up one folder.

Reply:
Good catch. I think I got it right this time...